opencti-platform/opencti-front/src/private/components/data/import/ImportWorkbenchesContent.tsx

```diff
@@ -54,14 +54,6 @@ export const workbenchLineFragment = graphql`
       creator {
         name
       }
-      entity_id
-      entity {
-        ... on StixObject{
-          representative {
-            main
-          }
-        }
-      }
     }
     works {
```

Comment:
do we really need works here ? since it's in the fragment FileWork_file ?

Reply:
Fixed + multiple improvement around relay